fourkitchens/suhumsci
# shs-4960: ck5 blockquote style adjustment

## Summary
This adjusts the blockquote styles for CK5. This issue is similar to other CK5 style issues where we need to compensate for `rems`.

## Steps to Test
1. Create a flexible page. Inside the text area, write some text, highlight it and turn it into a blockquote.
2. It should not be abnormally large and should look like the current CK version.
